src/extension.ts

```
@@ -127,6 +125,44 @@ function initAsyncCommand(context: vscode.ExtensionContext, commandId: string, c
 	}));
 }
 
+async function update(): Promise<void> {
+	if (lastOpenedDocumentType === DocumentType.Mongo) {
+		await MongoCommands.updateDocuments(connectedDb, lastCommand, lastOpenedMongoDocument);
+	}
+	else if (lastOpenedDocumentType === DocumentType.DocDB) {
+		await CosmosDBCommands.updateDocDBDocument(lastOpenedDocDBDocument);
+	}
+}
+
+async function updateOpenDocumentIfChanged(): Promise<void> {
+	let oldDocument, newDocument, recentDoc;
+	if (lastOpenedDocumentType === undefined) {
+		return;
+	}
+	try {
+		recentDoc = (lastOpenedDocumentType === DocumentType.DocDB) ? lastOpenedDocDBDocument : lastOpenedMongoDocument;
+		oldDocument = recentDoc.data;
+		const editor = vscode.window.activeTextEditor;
+		newDocument = JSON.parse(editor.document.getText());
+	}
+	catch {
+		return;
```

Why are we ignoring errors here?